src/App/Components/PasswordReset/ForgotPassword.js

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react';
+import axiosInstance from '../../../../configs/axios';
```

Comment:
The config folder is too far to reach, don't you think so? @nkpremices

Reply:
I do @truestbyheart 
Let me put it in the `src` folder